src/session-feedback/session-feedback.service.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { PartnerAccessEntity } from 'src/entities/partner-access.entity';
+import { PartnerEntity } from 'src/entities/partner.entity';
+import { SessionFeedbackEntity } from 'src/entities/session-feedback.entity';
+import { SessionEntity } from 'src/entities/session.entity';
+import { SubscriptionUserEntity } from 'src/entities/subscription-user.entity';
+import { SubscriptionEntity } from 'src/entities/subscription.entity';
+import { TherapySessionEntity } from 'src/entities/therapy-session.entity';
+import { UserEntity } from 'src/entities/user.entity';
+import { SessionFeedbackService } from 'src/session-feedback/session-feedback.service';
+import { SessionService } from 'src/session/session.service';
+import { FEEDBACK_TAGS_ENUM } from 'src/utils/constants';
+import { mockSessionEntity } from 'test/utils/mockData';
+import { Repository } from 'typeorm';
+
+const sessionFeedbackDto = {
+  sessionFeedbackId: 'session-feedback-id',
+  sessionId: mockSessionEntity.id,
+  feedbackTags: FEEDBACK_TAGS_ENUM.INSPIRING,
+  feedbackDescription: 'feedback-description',
+};
+
+describe('SessionFeedbackService', () => {
+  let service: SessionFeedbackService;
+  let mockPartnerAccessRepository: DeepMocked<Repository<PartnerAccessEntity>>;
+  let mockPartnerRepository: DeepMocked<Repository<PartnerEntity>>;
```

Comment:
Same comment/question as below about all the transitive dependencies for this service